### PR TITLE
Detroit GA4 Tracking

### PR DESCRIPTION
**Pull request by ColinBuyck**
# Pull Request Template

## Issue Overview

This PR addresses #1512 

- [x] This change addresses the issue in full
- [ ] This change addresses only certain aspects of the issue
- [ ] This change is a dependency for another issue
- [ ] This change has a dependency from another issue

## Description

This PR follows the pattern established by the HBA GA4 tracking script (https://github.com/housingbayarea/bloom/pull/565/files#diff-67d84b5cc47bada8f793de8a67aed99ccc66f9f7656fb474b93d2005fa3817c1)

## How Can This Be Tested/Reviewed?

This can be tested by ensuring that the previous UA script is in-place in the inspect element "Elements" tab along with:
<script async src="https://www.googletagmanager.com/gtag/js?id=G-92LSKY0KHG"></script>
<script> window.dataLayer = window.dataLayer || []; function gtag(){dataLayer.push(arguments);} gtag('js', new Date()); gtag('config', 'G-92LSKY0KHG'); </script>

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have added QA notes to the issue with applicable URLs
- [x] I have performed a self-review of my own code
- [ ] I have reviewed the changes in a desktop view
- [ ] I have reviewed the changes in a mobile view
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] I have assigned reviewers
- [ ] I have run `yarn generate:client` and/or created a migration if I made backend changes that require them
- [ ] I have exported any new pieces added to ui-components
- [ ] My commit message(s) is/are polished, and any breaking changes are indicated in the message and are well-described
- [ ] Commits made across packages purposefully have the same commit message/version change, else are separated into different commits

## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.

### CODE REVIEW

File: sites/public/.env.template [hunk 1]
```diff
@@ -13,7 +13,7 @@ CLOUDINARY_CLOUD_NAME=exygy
 CACHE_REVALIDATE=60
 NEW_RELIC_APP_NAME=Bloom Public
 NEW_RELIC_LICENSE_KEY=
-# this GTM key is for local development only
-GTM_KEY=GTM-KF22FJP
+GA_KEY=G-7N2S5HXLQE
```
Comment (emilyjablonski) [inline sites/public/.env.template:16]: This new key is also just for dev purposes, yeah?

Comment (ColinBuyck) [reply sites/public/.env.template:16]: Good point! That is the key for Detroit's prod sites so seems best to remove it from the template if you agree?

Comment (emilyjablonski) [reply sites/public/.env.template:16]: Yep, we wouldn't want to end up in a place where we're changing prod keys locally / have any prod keys in the template.

Comment (ColinBuyck) [reply sites/public/.env.template]: Makes sense! It's been removed 🥦

**Review by emilyjablonski — APPROVED** [review]

**Pull request merged**